CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # Changelog
```

Comment:
Updating changelog with linting fixes to see if it fixes the below

<img width="649" alt="Screenshot 2024-01-31 at 4 10 17 PM" src="https://github.com/MetaMask/design-tokens/assets/8112138/5c23b489-79e8-4bf3-a66b-58a3475a0d04">